sphinx/builders/latex/__init__.py

```diff
@@ -176,14 +176,15 @@ def init_context(self) -> None:
             key = (self.config.latex_engine, self.config.language[:2])
             self.context.update(ADDITIONAL_SETTINGS.get(key, {}))
 
-        # Apply extension settings to context
-        self.context['packages'] = self.usepackages
-
         # Apply user settings to context
         self.context.update(self.config.latex_elements)
         self.context['release'] = self.config.release
         self.context['use_xindy'] = self.config.latex_use_xindy
 
+        # Merge extensions' LaTeX package list to user settings
+        self.context['packages'] = (list(self.context['packages']) +
+                                    self.app.registry.latex_packages)
```

Comment:
I see user specified packages will come first. Maybe document this (see previous comment)